synapse/datamodel.py

```diff
@@ -558,14 +559,15 @@ def getModelDefs(self):
         mdef['forms'] = [f.getFormDef() for f in self.forms.values()]
         mdef['univs'] = [u.getPropDef() for u in self.univs.values()]
         mdef['tagprops'] = [t.getTagPropDef() for t in self.tagprops.values()]
+        mdef['interfaces'] = list(self.ifaces.items())
         return [('all', mdef)]
 
     def getModelDict(self):
         retn = {
             'types': {},
             'forms': {},
             'tagprops': {},
-            'interfaces': {},
+            'interfaces': list(self.ifaces.items())
```

Comment:
Why is ifaces a list and not a dictionary?

Reply:
ok. Added a small test as well.